pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -317,6 +317,23 @@ def on_validation_end(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningModul
             return
         self.save_checkpoint(trainer)
 
+    def on_train_end(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningModule') -> None:
+        """
+        Save a checkpoint when training stops.
+
+        This will only save a checkpoint if `save_last` is also enabled as the monitor metrics logged during
+        training/validation steps or end of epochs are not guaranteed to be available at this stage.
+        """
+        if self._should_skip_saving_checkpoint(trainer):
+            return
+        if self.save_last and self.verbose:
+            rank_zero_info("Saving latest checkpoint...")
```

Comment:
```suggestion
        if self._should_skip_saving_checkpoint(trainer) or not self.save_last:
            return
        if self.verbose:
            rank_zero_info("Saving latest checkpoint...")
```

Reply:
This is not necessary as we have the check below but I guess it doesn't hurt